src/data/grammar/grammar.json

```diff
@@ -32,13 +36,13 @@
         "PICTURES: #pro.capitalize#'s [_cosplayAdj:amazing,convincing,elaborate,unconventional]#_cosplayAdj# cosplay of [_who:#cardAlone#,#character#]#_who#",
         "New preview card from #teaserCardSet#! Is it a #descriptor# #realPowerfulCard#?"
     ],
-    "announcement": [
+    "announcement": [     
```

Comment:
There's a few of these random whitespaces that seem incorrect throughout the diff, it'd be easier to read if they weren't here